### ISSUE TITLE
Add ability to pause moving image background in Special Collections

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1385]
Not necessary for the Tuesday deploy to production, but we should have a way to pause the moving image background in Special Collections, for accessibility purposes.

**Comment by jasoncorum** [comment]
@caseyedavis12 - how does this look? i elected to not use a traditional pause button since that is being used for carousels. i wanted to better delineate it from the carousels. the text switches to "Play Video" on click.

![screen shot 2017-11-03 at 10 43 55 am](https://user-images.githubusercontent.com/3814190/32379581-40df57be-c084-11e7-9b39-1b8e49f79388.png)

**Comment by jasoncorum** [comment]
@caseyedavis12 - Did you have any thoughts on this? Here's another treatment I thought of while at Samvera Connect last. Do you like either? Other variations you'd like to see?

![screen shot 2017-11-15 at 11 28 24 am](https://user-images.githubusercontent.com/3814190/32847509-35614244-c9f8-11e7-9caf-47c9426cc985.png)

**Comment by caseyedavis12** [comment]
@jasoncorum: Could this be put in a corner on the screen and then brought to center on mobile?

**Comment by caseyedavis12** [comment]
@jasoncorum: actually we wouldn't need it on mobile, right?

**Comment by jasoncorum** [comment]
@caseyedavis12 - correct on mobile. i'll get a couple new screenshots.

**Comment by jasoncorum** [comment]
@caseyedavis12 - How's this? It stays in the top-right corner and disappears for mobile when we switch from video to image.

![screen shot 2017-11-16 at 9 58 14 am](https://user-images.githubusercontent.com/3814190/32897924-cbdc4518-cab4-11e7-9727-c00fa80ab6dc.png)

**Comment by caseyedavis12** [comment]
@jasoncorum perfect
________________________________
